README.rst

```diff
@@ -82,6 +82,21 @@ On a Debian based system, the installation is as follows::
     make && make install
     cd ..
 
+Installing the PostgresSQL session_exec extension
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is optional, but required if you choose to install the python packages
+into a virtualenv.
+
+::
+    git clone https://github.com/okbob/session_exec
```

Comment:
You're right. This is why reedstrm's fork exists. I added a blob under the code block to note the version difference.